addons/dotnet/dotnet.addon

```diff
@@ -0,0 +1,10 @@
+# Name: dotnet
+# Description: ImageStreams and Templates for developing and deploying application using .NET Core
+# URL: https://github.com/redhat-developer/s2i-dotnetcore
+
+oc apply --namespace openshift -f dotnet_imagestreams.json
+oc apply --namespace openshift -f dotnet_imagestreams_centos.json
```

Comment:
`dotnet_imagestreams_centos.json` and `dotnet_imagestreams.json` define the same image streams (CentOS vs RHEL based). Probably you don't want to import both?